app/models/presentation.rb

```diff
@@ -3,12 +3,14 @@ class Presentation < ApplicationRecord
   belongs_to :event
 
   validates :topic,     presence: true
-  validates :duration,  presence: true, numericality: true
   validates :person,    presence: true, unless: :presenter
   validates :presenter, presence: true, unless: :person
   validates :event,     presence: true
+  validates :duration,  presence: true, numericality: {
```

Comment:
What do you think about providing a custom error message to make it a little more clear why we might throw this error?